Yona/Yona/Others/EnvironmentManager.m

```diff
@@ -25,10 +25,10 @@ @implementation EnvironmentManager
 + (NSString *)baseUrlString
 {
     NSString *url = [[NSUserDefaults standardUserDefaults] stringForKey:@"YONA_URL"];
-    if (url == nil){
-//        url = @"https://beta.prd.yona.nu/";
-       // url = @"https://app.prd.yona.nu/";
-        url = @"https://app.beta.yona.nu/";
+        if (url == nil || [url isEqual:@""]){
+            url = @"https://app.beta.yona.nu/";   // test enviroment
+            // url = @"https://app.prd.yona.nu/"; // Production enviroment
+        //url = @"https://app.beta.yona.nu/";
```

Comment:
This line and the next one are inconsistently indented in comparison with the above ones